### ISSUE TITLE
bwa parameter change

### ISSUE THREAD

**Issue by ViriatoII** [ArimaGenomics/mapping_pipeline#7]
Hello,

I downloaded your fantastic pipeline last year and it seems to be slightly different now. 

I noticed that my version of the pipeline does bwa with a parameter -B 8 , which is not the case in your current version. 

Did you change this? If so, what is the relevance of it towards the Hi-C scaffolding?  It seems this parameter makes a more conservative mapping of the Hi-C reads, thus leading to more conservative scaffolding.

If you never had this parameter, than I am investigating a murder in which I am also the murderer, which is often the case with programming :)

Thank you,
Ricardo

**Comment by xz-000** [comment]
Good catch! We removed the parameter "-B 8" in our mapping pipeline, and used the default value implicitly ("-B 4"). 

We tested the performance of using "-B 4", "-B 6" and "-B 8" respectively, and found that the default value performs best in terms of variant calling and genotyping. Actually the difference is subtle. That is why we ended up using the default value. Thanks for bringing this up!

Best,
Arima Customer Support Team
